daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1250,18 +1252,21 @@ private[lf] object SBuiltin {
       }
     }
 
-    private[this] def executeCommit(args: util.ArrayList[SValue], machine: Machine): Unit =
-      machine.ptx
+    private[this] def executeCommit(
+        args: util.ArrayList[SValue],
+        machine: Machine,
+        onLedger: OnLedger): Unit =
```

Comment:
```suggestion
        onLedger: OnLedger,
    ): Unit =
```

Reply:
You really love your trailing commas :slightly_smiling_face: